app/celer-dump-data.cc

```diff
@@ -78,25 +81,74 @@ void print_particles(ParticleParams const& particles)
 /*!
  * Print element properties.
  */
-void print_elements(std::vector<ImportElement>& elements)
+void print_elements(std::vector<ImportElement>& elements,
+                    std::vector<ImportIsotope>& isotopes)
 {
     CELER_LOG(info) << "Loaded " << elements.size() << " elements";
     cout << R"gfm(
 # Elements
 
-| Element ID | Name | Atomic number | Mass (AMU) |
-| ---------- | ---- | ------------- | ---------- |
+| Element ID | Name | Atomic number | Mass (AMU) | Isotopes                                 | Relative abundance                                           |
+| ---------- | ---- | ------------- | ---------- | ---------------------------------------- | ------------------------------------------------------------ |
 )gfm";
 
     for (unsigned int element_id : range(elements.size()))
     {
         auto const& element = elements[element_id];
+
+        auto const labels = to_string(
+            join(element.isotopes_fractions.begin(),
+                 element.isotopes_fractions.end(),
+                 ", ",
+                 [&](auto const& key) { return isotopes[key.first].name; }));
+
+        auto const rel_abundances = to_string(join(
```

Comment:
Out of curiosity, is the `to_string` needed? It should be able to just reuse this "streamable" object.